### ISSUE TITLE
pyxel: No module named 'pyxel.cli'

### ISSUE THREAD

**Issue by blinry** [NixOS/nixpkgs#359836]
## Describe the bug

When building pyxel on the current master (cee3b715318cc3d748862b0736718ac69f73db53), and running the resulting binary, I get the following error:

```
Traceback (most recent call last):
  File "/nix/store/835662msvvs4am7ndqpv0y5zf3nmpqyz-pyxel-2.1.6/bin/.pyxel-wrapped", line 6, in <module>
    from pyxel.cli import cli
ModuleNotFoundError: No module named 'pyxel.cli'
```

## Steps To Reproduce

1. `nix run nixpkgs#pyxel`

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.1, NixOS, 25.05 (Warbler), 25.05.20241125.4633a7c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/store/frfyxcpzsdasdin76x83krbhpgkis8b0-source`

## Notify maintainers

@TomaSajt

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by TomaSajt** [comment]
Fixed in maturin 1.7.7

**Comment by cybardev** [comment]
I have this `shell.nix` that doesn't work:
```nix
{
  pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/nixos-24.11.tar.gz") { },
}:
pkgs.mkShell {
  nativeBuildInputs = with pkgs.buildPackages; [
    python313Full
    pyxel
  ];
}
```
And this one that does (but i don't want cuz of unstable):
```nix
{
  pkgs ? import (fetchTarball "https://nixos.org/channels/nixos-unstable/nixexprs.tar.xz") { },
}:
pkgs.mkShell {
  nativeBuildInputs = with pkgs.buildPackages; [
    python313Full
    pyxel
  ];
}
```

What's the recommended way for using `pyxel` in `24.11`?
